ci/run_python_tests.sh

```diff
@@ -51,6 +54,9 @@ if [ -z "${API_ZEN_REPO_URL:-}" ]; then
   fn_die "Error: API_ZEN_REPO_URL variable is not set. Exiting ..."
 fi
 
+if [ -z "${MAINTAINER_KEYS:-}" ]; then
```

Comment:
Keep it in mind that there is ONE existing `MAINTAINER_KEYS` var set under **setup_env.sh** file. If this is what you need here then bingo IF not then change this name and add value under Travis CI config UI

Reply:
that's not what I need, I added a new env var and this is an outdated version you are checking, its name is ZEN_REPO_MAINTAINER_KEYS